src/scripts/main.js

```diff
@@ -1,3 +1,19 @@
 'use strict';
 
-// write your code here
+const data = [...document.querySelectorAll('.population')];
+
+const populations = data.map((value) => {
+  const text = value.innerText.trim();
+  const numericText = text.split(',').join('');
+
+  return Number(numericText);
+});
+
+const total = populations.reduce((sum, value) => sum + value, 0);
```

Comment:
Check if 'populations' array is not empty before performing the reduce operation to avoid potential errors.